### PR TITLE
Hookup make url, shared access signature and account information

### PR DESCRIPTION
**Pull request by rakshith91**

### CODE REVIEW

File: azure-storage-blob/azure/storage/blob/blob_service_client.py [hunk 1]
```diff
@@ -88,13 +121,63 @@ def generate_shared_access_signature(
             ip=None,  # type: Optional[str]
             protocol=None  # type: Optional[str]
         ):
-        pass
+        '''
+        Generates a shared access signature for the blob service.
+        Use the returned signature with the sas_token parameter of any BlobService.
+        :param ResourceTypes resource_types:
+            Specifies the resource types that are accessible with the account SAS.
+        :param AccountPermissions permission:
```
Comment (bryevdv) [inline azure-storage-blob/azure/storage/blob/blob_service_client.py]: I had usually seen the `type` specified separately, which is kind of cumbersome. Do you have a link to any documentation for this format of `param` with the type and name together? I'm used to using Google style doctring syntax, and ironically actual documentation on this "low level" docstring format seems hard to come by.

File: azure-storage-blob/azure/storage/blob/blob_service_client.py [hunk 2]
```diff
@@ -78,7 +88,30 @@ def create_configuration(**kwargs):
...
+        new_scheme = protocol or parsed_url.scheme
+        query = []
+        if sas_token:
+            query.append(sas_token)
+        new_url = "{}://{}{}".format(
+            new_scheme,
+            parsed_url.netloc,
+            parsed_url.path)
+        if query:
+            new_url += "?{}".format('&'.join(query))
```
Comment (bryevdv) [inline azure-storage-blob/azure/storage/blob/blob_service_client.py]: `query` is only ever empty or just has `sas_token` correct? Unless there's a strong likelihood of this needing generalization later, I'd consider simplifying:
```
if sas_token:
    new_url += "?{}".format(sas_token)
```
and delete the three lines above starting at `query = []`

File: azure-storage-blob/azure/storage/blob/blob_service_client.py [hunk 3]
```diff
@@ -88,13 +121,63 @@ def generate_shared_access_signature(
...
+            raise ValueError('Account key should not be None')
+
+        sas = SharedAccessSignature(self.account_name, self.account_key)
+        return sas.generate_account(Services.BLOB, resource_types, permission,
+                                    expiry, start=start, ip=ip, protocol=protocol)
 
     def get_account_information(self, timeout=None):
         # type: (Optional[int]) -> Dict[str, str]
         """
         :returns: A dict of account information (SKU and account type).
```
Comment (bryevdv) [inline azure-storage-blob/azure/storage/blob/blob_service_client.py]: As a user I would be happy if this docstring listed the explicit key values that can be expected, or gave an example output

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Pull request merged**